### PR TITLE
v3.5: date/time fields

### PR DESCRIPTION
**Pull request by distantnative**

### CODE REVIEW

File: content/v35/3_date/page.txt [hunk 1]
```diff
@@ -7,5 +7,139 @@ Text:
...
+The fields now support a new `display` option which defines how the date is displayed in the Panel. Offering a range of options:
+
+```yaml
+# default
+display: YYYY-MM-DD
+
+# For ze Germäns
+display: DD.MM.YYYY
+
+# The Brtits
```
Comment (bastianallgeier) [inline content/v35/3_date/page.txt]: I'm not sure if this is a typo or intended 😂

File: content/v35/3_date/page.txt [hunk 2]
```diff
@@ -7,5 +7,139 @@ Text:
...
+
+```yaml
+display: DD.MM.YYYY
+```
+
+The field expects any input to match this format as well and converts it to a valid date internally. 
+
+```yaml
+# input -> converted
+29.01.1989 -> 29.01.1989
```
Comment (bastianallgeier) [inline content/v35/3_date/page.txt:62]: I might be blind, but are the two different?

File: content/v35/3_date/page.txt (hunk 1 above)
Comment (distantnative) [reply content/v35/3_date/page.txt]: Oh my 😂

File: content/v35/3_date/page.txt (hunk 2 above)
Comment (distantnative) [reply content/v35/3_date/page.txt:62]: No, the point was: if you enter it fully of course it gets parsed correctly but also with the following lines if you enter only partially

Comment (bastianallgeier) [reply content/v35/3_date/page.txt:62]: Ah, ok. I completely misunderstood this.

File: content/v35/3_date/page.txt (hunk 1 above)
Comment (distantnative) [reply content/v35/3_date/page.txt]: Fixed

File: content/v35/3_date/page.txt (hunk 2 above)
Comment (distantnative) [reply content/v35/3_date/page.txt:62]: Maybe you have an idea how to make this clearer?

Comment (bastianallgeier) [reply content/v35/3_date/page.txt:62]: I think it's fine. It was me :)

**Review by bastianallgeier — APPROVED** [review]

**Pull request merged**